cmake/UnitTest/test.rc

```diff
@@ -14,7 +14,7 @@ BEGIN
         BEGIN
                 BLOCK "000004b0"
                 BEGIN
-                        VALUE "CompanyName", "MultiMC Contributors"
+                        VALUE "CompanyName", "UltimMC Contributors"
```

Comment:
Nope, most of code written by MultiMC so here should be MultiMC contributors

Reply:
Technically all the code in MultiMC is also part of UltimMC, so UltimMC contributors includes MultiMC contributors?

Reply:
I guess no, people who contributed to MultiMC contributed to MultiMC bot UltimMC so they MultiMC contributors

Reply:
okay, will solve